lib/v2/hoyolab/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (router) => {
+    router.get('/news/:language/:gids/:type/:size?', require('./news'));
```

Comment:
https://github.com/DIYgod/RSSHub/blob/43c091aa2228c11e98b3a6666e1a11f6ffca2ab6/lib/v2/hoyolab/news.js#L69-L75

Since all three parameters have their default values which means they are optional parameters, the path needs to be changed as well as https://github.com/DIYgod/RSSHub/blob/43c091aa2228c11e98b3a6666e1a11f6ffca2ab6/website/docs/routes/game.mdx#L241 and https://github.com/DIYgod/RSSHub/blob/43c091aa2228c11e98b3a6666e1a11f6ffca2ab6/lib/v2/hoyolab/maintainer.js#L2

docs: https://docs.rsshub.app/joinus/new-rss/before-start#routerjs